lib/pluginmanager/install.rb

```diff
@@ -90,7 +91,13 @@ def install_gems_list!(install_list)
 
     # Add plugins/gems to the current gemfile
     puts("Installing" + (install_list.empty? ? "..." : " " + install_list.collect(&:first).join(", ")))
-    install_list.each { |plugin, version, options| gemfile.update(plugin, version, options) }
+    install_list.each do |plugin, version, options|
+      if preserve?
```

Comment:
maybe add a debug log message here to make the behavior known on a per plugin basis?